### PR TITLE
Hungarian translate update

### PR DESCRIPTION
**Pull request by ambrits**
Theoretically, I have reached the end with the integration of the Hungarian language. There may be typos or misinterpretations, but I will correct them later if I find them.

### CODE REVIEW

**Review by michaelherger — APPROVED** [review]

**Comment by michaelherger** [comment]
I'm sorry to spoil the party, but there are many more `strings.txt` files for each of the plugins in Slim/Plugin...

**Comment by michaelherger** [comment]
But thanks a ton for the work you've done already!

File: strings.txt [hunk 1]
```diff
@@ -13788,6 +13788,7 @@ OF
 	FI	/
 	FR	de
 	HE	מתוך
+	HU
```
Comment (michaelherger) [inline strings.txt:13791]: Should this really be empty?

File: strings.txt [hunk 2]
```diff
@@ -13807,6 +13808,7 @@ TO
 	FI	kohteeseen
 	FR	à
 	HE	אל
+	HU
```
Comment (michaelherger) [inline strings.txt:13811]: Empty on purpose?

**Comment by ambrits** [comment]
Yes. This cannot be interpreted in Hungarian. Then, during the testing, I see where it appears and then correct it if necessary.

Tamás

**Comment by ambrits** [comment]
No problem. As I have time, I will work on them as well. 

Tamás.

**Pull request merged**